src/main/java/roomescape/controller/TimeController.java

```diff
@@ -0,0 +1,38 @@
+package roomescape.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import roomescape.domain.dto.TimeRequest;
+import roomescape.domain.dto.TimeResponse;
+import roomescape.service.TimeService;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
+@RequestMapping("/times")
+public class TimeController {
+    private final TimeService timeService;
+
+    public TimeController(TimeService timeService) {
+        this.timeService = timeService;
+    }
+
+    @PostMapping
+    public ResponseEntity<TimeResponse> create(@RequestBody TimeRequest request) {
+        TimeResponse response = timeService.save(request);
```

Comment:
개인적으로 CRUD는 동사를 맞추는게 편한 것 같더라구요! 어떻게 생각하시나요? (Reservation에서도 `save`로 되어 있네요)

```suggestion
    @PostMapping
    public ResponseEntity<TimeResponse> save(@RequestBody TimeRequest request) {
        TimeResponse response = timeService.save(request);
```

Reply:
같은 동사로 하는 게 확실히 편할 것 같습니다! 수정하겠습니다 😄